lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -389,6 +390,8 @@ def _call(self, method: Callable, params: ApiParams, context: ApiContext):
             raise APIInternalError(f"problem in preprocess, missing {e}") from e
         try:
             # prepare payload for Elasticsearch query
+            elastic_user = self.config.get("Indexing", "elastic_user")
+            elastic_password = self.config.get("Indexing", "elastic_password")
```

Comment:
Does the intlab Opensearch instance support TLS and user authentication? If so, we'll need the credentials before we can deploy with this change. If not ... and maybe anyway ... should we consider implementing a fallback to `http` otherwise?

Also, I'm not enthusiastic about the `except Exception` below claiming an "assembly error" (subclass helper) if the actual problem is that we're requiring credentials that aren't defined. I suspect these should be in a separate try block or at least we should have a separate catch for the config errors.